app-backend/db/src/main/scala/StacExportDao.scala

```diff
@@ -91,12 +94,16 @@ object StacExportDao extends Dao[StacExport] {
       }
     }
 
-  def hasProjectViewAccess(layerDefinitions: List[StacExport.LayerDefinition],
-                           user: User): ConnectionIO[Boolean] =
-    layerDefinitions traverse { ld =>
-      ProjectDao.authProjectLayerExist(ld.projectId,
-                                       ld.layerId,
-                                       user,
-                                       ActionType.View)
-    } map { _.foldLeft(true)(_ && _) }
+  def hasProjectViewAccess(
+      annotationProjectId: UUID,
+      user: User
+  ): ConnectionIO[Boolean] =
+    for {
+      authProject <- AnnotationProjectDao.authorized(
+        user,
+        ObjectType.Project,
```

Comment:
@aaronxsu just pointed out to me that this should be `ObjectType.AnnotationProject`